source/core_mqtt.c

```diff
@@ -742,9 +742,10 @@ static int32_t recvExact( const MQTTContext_t * pContext,
             bytesRemaining -= ( size_t ) bytesRecvd;
             totalBytesRecvd += ( int32_t ) bytesRecvd;
             pIndex += bytesRecvd;
-            LogDebug( ( "BytesReceived=%ld, BytesRemaining=%lu, "
+            LogDebug( ( "BytesReceived=%ld, BytesRemaining=%lu, TotalBytesReceived=%ld.",
                         ( long int ) bytesRecvd,
-                        ( unsigned long ) bytesRemaining ) );
+                        ( unsigned long ) bytesRemaining,
+                        ( long int ) totalBytesRecvd ) );
```

Comment:
Wasn't this removed due to complexity, or is it fine now?

Reply:
It was but by keeping the string on the same line, the complexity is kept at 8. I'm working on using sed to remove the logs before checking complexity though. I don't think it is ideal to make logs less informative to satisfy complexity.